### PR TITLE
chore(generate): Additional link to PDP/Accelerate

### PR DESCRIPTION
**Pull request by janpio**
/integration

### CODE REVIEW

File: packages/cli/src/Generate.ts [hunk 1]
```diff
@@ -263,6 +263,12 @@ const prisma = new PrismaClient()`)}
 ${dim('```')}
 
 See other ways of importing Prisma Client: ${link('http://pris.ly/d/importing-client')}
+
+****************
+Deploying your app to Serverless or Edge Functions?
+Try Prisma Accelerate for Connection Pooling and Caching.
+${link('https://pris.ly/cli/accelerate')}
+****************
```
Comment (janpio) [inline packages/cli/src/Generate.ts:271]: TODO: Draw box around it

File: packages/cli/src/Generate.ts [hunk 2]
```diff
@@ -28,6 +29,7 @@ import resolvePkg from 'resolve-pkg'
 import { getHardcodedUrlWarning } from './generate/getHardcodedUrlWarning'
 import { breakingChangesMessage } from './utils/breakingChanges'
 import { simpleDebounce } from './utils/simpleDebounce'
+import { isModuleNamespaceObject } from 'util/types'
```
Comment (janpio) [inline packages/cli/src/Generate.ts:32]:
```suggestion
```

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 3]
```diff
@@ -56,6 +62,12 @@ describe('using cli', () => {
 
         See other ways of importing Prisma Client: http://pris.ly/d/importing-client
 
+        ┌─────────────────────────────────────────────────────────────┐
+        │  Deploying your app to Serverless or Edge Functions?        │
+        │  Try Prisma Accelerate for Connection Pooling and Caching.  │
```
Comment (janpio) [inline packages/cli/src/__tests__/commands/Generate.test.ts:67]:
```suggestion
        │  Deploying your app to serverless or edge functions?        │
        │  Try Prisma Accelerate for connection pooling and caching.  │
```

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 4]
```diff
@@ -598,28 +718,40 @@ describe('--schema from parent directory', () => {
 
         See other ways of importing Prisma Client: http://pris.ly/d/importing-client
 
-                                          `)
+        ┌─────────────────────────────────────────────────────────────┐
+        │  Deploying your app to Serverless or Edge Functions?        │
+        │  Try Prisma Accelerate for Connection Pooling and Caching.  │
```
Comment (janpio) [inline packages/cli/src/__tests__/commands/Generate.test.ts:723]:
```suggestion
        │  Deploying your app to serverless or edge functions?        │
        │  Try Prisma Accelerate for connection pooling and caching.  │
```

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 5]
```diff
@@ -539,26 +647,38 @@ describe('--schema from parent directory', () => {
...
```
Comment (janpio) [inline packages/cli/src/__tests__/commands/Generate.test.ts:652]:
```suggestion
        │  Deploying your app to serverless or edge functions?        │
        │  Try Prisma Accelerate for connection pooling and caching.  │
```

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 6]
```diff
@@ -483,26 +579,38 @@ describe('--schema from parent directory', () => {
...
```
Comment (janpio) [inline packages/cli/src/__tests__/commands/Generate.test.ts:584]:
```suggestion
        │  Deploying your app to serverless or edge functions?        │
        │  Try Prisma Accelerate for connection pooling and caching.  │
```

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 7]
```diff
@@ -327,6 +393,12 @@ describe('using cli', () => {
 
         See other ways of importing Prisma Client: http://pris.ly/d/importing-client
 
+        ┌─────────────────────────────────────────────────────────────┐
+        │  Deploying your app to Serverless or Edge Functions?        │
+        │  Try Prisma Accelerate for Connection Pooling and Caching.  │
```
Comment (janpio) [inline packages/cli/src/__tests__/commands/Generate.test.ts:398]:
```suggestion
        │  Deploying your app to serverless or edge functions?        │
        │  Try Prisma Accelerate for connection pooling and caching.  │
```

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 8]
```diff
@@ -305,6 +365,12 @@ describe('using cli', () => {
...
```
Comment (janpio) [inline packages/cli/src/__tests__/commands/Generate.test.ts:370]:
```suggestion
        │  Deploying your app to serverless or edge functions?        │
        │  Try Prisma Accelerate for connection pooling and caching.  │
```

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 9]
```diff
@@ -274,6 +328,12 @@ describe('using cli', () => {
...
```
Comment (janpio) [inline packages/cli/src/__tests__/commands/Generate.test.ts:333]:
```suggestion
        │  Deploying your app to serverless or edge functions?        │
        │  Try Prisma Accelerate for connection pooling and caching.  │
```

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 10]
```diff
@@ -254,6 +302,12 @@ describe('using cli', () => {
...
```
Comment (janpio) [inline packages/cli/src/__tests__/commands/Generate.test.ts:307]:
```suggestion
        │  Deploying your app to serverless or edge functions?        │
        │  Try Prisma Accelerate for connection pooling and caching.  │
```

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 11]
```diff
@@ -203,6 +239,12 @@ describe('using cli', () => {
...
```
Comment (janpio) [inline packages/cli/src/__tests__/commands/Generate.test.ts:244]:
```suggestion
        │  Deploying your app to serverless or edge functions?        │
        │  Try Prisma Accelerate for connection pooling and caching.  │
```

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 12]
```diff
@@ -170,6 +200,12 @@ describe('using cli', () => {
...
```
Comment (janpio) [inline packages/cli/src/__tests__/commands/Generate.test.ts:205]:
```suggestion
        │  Deploying your app to serverless or edge functions?        │
        │  Try Prisma Accelerate for connection pooling and caching.  │
```

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 13]
```diff
@@ -117,6 +135,12 @@ describe('using cli', () => {
...
```
Comment (janpio) [inline packages/cli/src/__tests__/commands/Generate.test.ts:140]:
```suggestion
        │  Deploying your app to serverless or edge functions?        │
        │  Try Prisma Accelerate for connection pooling and caching.  │
```

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 14]
```diff
@@ -598,28 +718,40 @@ describe('--schema from parent directory', () => {
...
+                \`\`\`
+                import { PrismaClient } from './generated/client/edge'
+                const prisma = new PrismaClient()
+                \`\`\`
+
+                See other ways of importing Prisma Client: http://pris.ly/d/importing-client
+
+                ┌─────────────────────────────────────────────────────────────┐
+                │  Deploying your app to Serverless or Edge Functions?        │
+                │  Try Prisma Accelerate for Connection Pooling and Caching.  │
```
Comment (janpio) [inline packages/cli/src/__tests__/commands/Generate.test.ts:750]:
```suggestion
                │  Deploying your app to serverless or edge functions?        │
                │  Try Prisma Accelerate for connection pooling and caching.  │
```

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 15]
```diff
@@ -539,26 +647,38 @@ describe('--schema from parent directory', () => {
...
+                import { PrismaClient } from './subdirectory/@prisma/client/edge'
+                const prisma = new PrismaClient()
+                \`\`\`
 
-        See other ways of importing Prisma Client: http://pris.ly/d/importing-client
+                See other ways of importing Prisma Client: http://pris.ly/d/importing-client
+
+                ┌─────────────────────────────────────────────────────────────┐
+                │  Deploying your app to Serverless or Edge Functions?        │
+                │  Try Prisma Accelerate for Connection Pooling and Caching.  │
```
Comment (janpio) [inline packages/cli/src/__tests__/commands/Generate.test.ts:677]:
```suggestion
                │  Deploying your app to serverless or edge functions?        │
                │  Try Prisma Accelerate for connection pooling and caching.  │
```

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 16]
```diff
@@ -428,26 +512,38 @@ describe('--schema from project directory', () => {
...
+                import { PrismaClient } from './@prisma/client/edge'
+                const prisma = new PrismaClient()
+                \`\`\`
 
-        See other ways of importing Prisma Client: http://pris.ly/d/importing-client
+                See other ways of importing Prisma Client: http://pris.ly/d/importing-client
+
+                ┌─────────────────────────────────────────────────────────────┐
+                │  Deploying your app to Serverless or Edge Functions?        │
+                │  Try Prisma Accelerate for Connection Pooling and Caching.  │
```
Comment (janpio) [inline packages/cli/src/__tests__/commands/Generate.test.ts:542]:
```suggestion
                │  Deploying your app to serverless or edge functions?        │
                │  Try Prisma Accelerate for connection pooling and caching.  │
```

**Review by Jolg42 — APPROVED** [review]
Looks good to me ✔️ 

Note: It makes the output longer, some people will probably not like it because they considered it noisy already. But that's a different problem we need to improve another day.

**Pull request merged**